docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -223,17 +231,17 @@ To enable SNI, use the following property:
 quarkus.tls.key-store.sni=true # Disabled by default
 ----
 
-With this setting enabled, the client indicate the server name during the TLS handshake, allowing the server to select the right certificate:
+With this setting enabled, the client indicates the server name during the TLS handshake, allowing the server to select the right certificate:
 
-- When configuring the keystore with PEM files, multiple CRT/Key must be given.
-- When configuring the keystore with a JKS or a P12 file, it selects one alias based on the SNI hostname. In this case, all the keystore password and alias password must be the same. Do not set the `alias` property in this case.
+* When configuring the keystore with PEM files, multiple CRT or Key must be given.
```

Comment:
Update:

```
With this setting enabled, the client indicates the server name during the TLS handshake, allowing the server to select the appropriate certificate:

* When configuring the keystore with PEM files, multiple certificate (CRT) and key files must be provided.
CRT is a common file extension for X.509 certificate files, typically in PEM (Privacy-Enhanced Mail) format.
These files contain the public certificate.
* When configuring the keystore with a JKS or P12 file, the server selects one alias based on the SNI hostname.
In this case, all keystore and alias passwords must be identical, and there is no need to set the alias property.
```

Is this better now @jmartisk @sheilamjones ?

Reply:
LGTM @MichalMaler. 
Just a small thing, I checked online and there isn't a hyphen in the term "Privacy Enhanced Mail". 
Also, should it be spelled in full in the first sentence instead of the second, that is, in the sentence  "When configuring the keystore with Privacy Enhance Mail (PEM) files......."